backend/server/database.py

```diff
@@ -200,7 +198,7 @@ def overwrite_all():
     overwrite_archives()
 
 def optionally_create_new_data():
-    # TODO: DELETE NEXT LINE IN PROD
+    # TODO: DELETE NEXT LINES IN PROD
```

Comment:
Is this still relevant?

Reply:
Yes, I want that for now, this gets overwritten every time you relaunch your app, but this is not desirable in prod